src/components/MobileNavbar/MobileNavbar.js

```diff
@@ -0,0 +1,93 @@
+import '../MobileNavbar/MobileNavbar.scss';
+
+import Brightness6RoundedIcon from '@mui/icons-material/Brightness6Rounded';
+import { IconButton } from '@mui/material';
+import React from 'react';
+import { useState } from 'react';
+
+import logo_icon_dark_theme from '../../assets/images/logo_icon_dark_theme.png';
+import logo_icon_light_theme from '../../assets/images/logo_icon_light_theme.png';
+import logo_text_dark_theme from '../../assets/images/logo_text_dark_theme.png';
+import logo_text_light_theme from '../../assets/images/logo_text_light_theme.png';
+
+export function MobileNavbar() {
+  const [theme, setTheme] = useState(false);
+
+  function switchTheme() {
```

Comment:
1. Ustawianie na początku funkcji `setTheme(false)` a zaraz poniżej sprawdzanie czy jest `false`, nie jest najlepszym podejściem, wiem że działa, ale może się tak zdażyć że czasem nie zadziała. Dodatkowo wyglada nieczytelenie. Proponuję przenieść do `setTheme(false)` na koniec `else`
2. `if (theme == false)` - zawsze stosujemy 3 znaki porównania w if `===` ale tutaj możemy to jeszcze bardziej uprosić i napisać `if(!theme)`
3. Staramy się nie zmianiać wartości JSX za pomocą` document.getElementsByClassName...` robimy to raczej w ostateczności. Staramy się to robić bezpośrednio w JSX.
np. dynamiczna zmiana koloru ikony:  
` <Brightness6RoundedIcon sx={{ color: theme ? "#FFFFFF" : "#000000"}}/>`
np. dynamiczna zmiana koloru elementu: 
`<div className={theme ? TextLight : TextDark}>TUSK TUSK TUSK</div>`
można też zmieniać bezpośrednio css ale lepiej sobie przygotować 2 osobne klasy